docs/source/posts/2022/2022-05-25-pre-gsoc-journey-praneeth.rst

```diff
@@ -0,0 +1,44 @@
nts due to which I found out that for GSoC we have to select the organizations then apply and this would be the basic flow of the program. So taking this into consideration I started searching for organizations on GSoC archives. Firstly I was looking for any Javascript related project as I had a better command on it than python and there I found out about p5.js which was a creative coding library. I forked their repo, cloned it, and then tried to build it on my local machine but after many attempts I was still unsuccessful to build it locally. I tried to contact, regarding this issue with the community and also mailed an old GSoC student but they too were unable to solve the query. This disappointed me a lot and kind of I was going to end my opensource journey here, but just thought of giving it a try again. While searching through the organizations I found lots of known projects regarding which I had no idea that those too were open source. While searching I found Python Software Foundation in the organization list and thought why not to consider python. After looking into the organization I found out that PSF was an umbrella project for many sub projects in which FURY(Free Unified Rendering in pYthon) suited me the most because I was a beginner and didn’t had any idea regarding the data analysis, API formation and the other stuff. But because of prior experience and interest in Game Dev, I was comfortable with FURY. I again forked, cloned and tried to execute the first tutorial on it, it took few minutes to run then few more minutes to download some required files and then a black box appeared but with nothing on it, I thought there would be some error with something so re executed it and it took few minutes and the first Earth Animation tutorial was successfully executed. I was fascinated by viewing the visuals of the Earth with the orbiting moon and the satellite animation (I had doubts whether my laptop will be able to handle these computations or not, but till now It hasn’t proved me wrong in most of the cases). I checked the code then and within 200 lines of code all this was all possible. That made me understand the real power of FURY. After this I checked for some of the good first issues but as it was GSoC period most of the issues were under process so I tried to learn more about the tutorials and the code. There I found a few documentation errors which I fixed as my first PR. I also pointed out some issues which I faced while working with FURY. FURY has a great discord community which engages newbies and solves most of their issues. After two months GSoC Contributors Applications started and I also tried to apply for the same. I created a proposal on Improving the UI system but due to lack of experience and as I was unable to prove my coding skills, I was rejected. But this didn’t disengage me from FURY. I loved the simplicity by which each part was created. Then as time passed I worked on some of the issues and created some more PRs.
```

Comment:
`I tried to contact, regarding this issue with the community and also mailed an old GSoC student but they too were unable to solve the query` you can try to rewrite it like this: I tried to contact them regarding this issue and also mailed an old GSoC student but they too were unable to solve the query